guides/common/modules/proc_synchronizing-the-system-clock-with-chronyd.adoc

```diff
@@ -1,6 +1,6 @@
 [id="synchronizing-the-system-clock-with-chronyd_{context}"]
 
```

Comment:
```suggestion
```

Reply:
Fixed. I should probably run a sed to fix this in all the files -- separate PR, of course.